non-oss/pharos_pro/addons/kontena-universal-lb/addon.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+Pharos.addon 'kontena-universal-lb' do
+  version '0.1.0'
+  license 'Kontena License'
```

Comment:
Might make sense to configure tolerations and node selectors? (can be follow-up pr)

Reply:
I'd do it in a follow-up PR later on when we know how the LB daemonset will get their selectors & taints.